benchmark/README.md

```diff
@@ -0,0 +1,15 @@
+# Running Performance Tests
```

Comment:
Since these are perf tests how about we name the folder `performance`. 

And actually - I feel like maybe we should just put this under the test folder - so `test/Performance`

Reply:
Changed folder name to `performance`. BenchmarkDotnet needs to be run from it's own console project so I can't add this to the Tests project. We could nest the current Tests project into another folder and move performance under test so we would have `test/Performance` and `test/<current-tests-project>`